applets/009-network/applet.py

```diff
@@ -0,0 +1,80 @@
+# usage: exec 009-network setup:direct host1 host2 host3 ...
+# obviously, also the physical replug hast to happen as well
+
+import os
+import subprocess
+import time
+
+from threading import Thread
+
+LOCAL_CONT_PATH = os.path.dirname(os.path.realpath(__file__))
+REMOTE_CONT_PATH = "/tmp/net-applet-shuffler"
+CONTR_NAME = "network-controller.py"
+
+
+class ControllerStart(Thread):
+
+    host_user = "host_user"
+    host_ip = "host_ip"
+    arguments = "arguments"
+
+    def __init__(self, user, ip, arguments):
+        Thread.__init__(self)
+        self.daemon = True
+        self.host_user = user
+        self.host_ip = ip
+        self.arguments = arguments
+
+    def run(self):
+        ssh_command = "ssh {}@{} sudo python3.5 {}/{} {}".format(
```

Comment:
should be python3 to be compatible with older and future releases. There is no specific feature required to bound to 3.5.